eventhandlers.go

```diff
@@ -45,7 +45,7 @@ func HandleGetSliTriggeredEvent(ddKeptn *keptnv2.Keptn, incomingEvent cloudevent
 	// The get-sli.started cloud-event is new since Keptn 0.8.0 and is required to be send when the task is started
 	_, err := ddKeptn.SendTaskStartedEvent(data, ServiceName)
 	if err != nil {
-		errMsg := fmt.Sprintf("Failed to send task started CloudEvent (%s), aborting...", err.Error())
+		errMsg := fmt.Errorf("failed to send task started CloudEvent (%s), aborting... ", err.Error())
```

Comment:
In these situations, here and the other changes, do not be afraid to use `err` and reassign to it the new value (or simply rename errMsg to err as Errorf returns a value of type error).

To not lose the previous error in the reassignment process use the %w (wrap) instead of (%s) and pass err instead of err.Error()

For example, this line becomes
`err = fmt.Errorf("failed to send task started CloudEvent (%w), aborting... ", err)`